sdk/src/client/secret/ledger_nano.rs

```diff
@@ -544,6 +544,7 @@ fn merge_unlocks(
                     merged_unlocks.push(Unlock::Reference(ReferenceUnlock::new(*block_index as u16)?));
                 }
                 Address::Nft(_nft) => merged_unlocks.push(Unlock::Nft(NftUnlock::new(*block_index as u16)?)),
+                _ => todo!("What do we do here?"),
```

Comment:
ID say also a new issue for unlock with Multi, dont think its described somewhere yet, but i bet itll be a MultiUnlock type :D

Reply:
https://github.com/iotaledger/iota-sdk/issues/1354